geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -79,7 +78,7 @@ The Redis API for <%=vars.product_name%> currently supports the following comman
 -   **Sets**: SADD, SMEMBERS, SREM
 -   **Strings**: APPEND, DECR, GET, GETRANGE, INCR, INCRBY, INCRBYFLOAT, MGET, SET, SETNX, STRLEN
 
-The following Redis API for <%=vars.product_name%> commands are **unsupported**. Unsupported
+The following Redis commands are **unsupported** by <%=vars.produce_name%>'s API for Compatibility with Redis. Unsupported
 commands are available to use, but have not been fully tested. There is no guarantee they will work
 exactly as expected.
 
```

Comment:
@nonbinaryprogrammer & @ringles Should we remove this content?

Reply:
I left everything related to unsupported commands because I knew Ray and Hutch were going to take all that out. We'll probably have some annoying merge conflicts though

Reply:
Sounds good.  Thank you!